app/api/auth.py

```diff
@@ -17,55 +17,48 @@
 bp = Blueprint('auth', __name__, url_prefix='/auth')
 
 
-@bp.route('/register', methods=['POST'])
-def register():
-    if not request.json:
-        abort(500)
-    name = request.json.get("name", None)
-    username = request.json.get("username", None)
-    password = request.json.get("password", None)
-    if not name or not username or not password:
-        return jsonify({"msg": "Invalid Request"}), 400
-
-    user = mongo.db.users.count({
-        "username": username
-    })
-    if user > 0:
-        return jsonify({"msg": "Username already taken"}), 500
-
-    id = mongo.db.users.insert_one({
-        "name": name,
-        "password": pbkdf2_sha256.hash(password),
-        "username": username
-    }).inserted_id
-    return jsonify(str(id))
-
+@bp.route('/login', methods=['GET','POST'])
+@bp.route('/login/<string:id>', methods=['PUT'])
+def login(id):
 
-@bp.route('/login', methods=['POST'])
-def login():
-    if not request.json:
-        abort(500)
-
-    username = request.json.get('username', None)
-    password = request.json.get('password', None)
-    if not username:
-        return jsonify(msg="Missing username parameter"), 400
-    if not password:
-        return jsonify(msg="Missing password parameter"), 400
+    if request.method == "GET":
+       username = request.json.get("username")
+       password = request.json.get("password")   
+       URL = 'https://hr.excellencetechnologies.in/attendance/API_HR/api.php'
+       payload = {'username': username, "password": password, "action": "login", "token": None}
+       response = requests.get(url=URL, json=payload)
+       return jsonify(response.json())
+    
+    if request.method == "POST":

```

Comment:
what are there two request types in this GET and POST?? what is the logic behind this. 

frontend will just call one login api as already told on slack. so they why two are here again??